### DISCUSSION TITLE
PaddleOCR 是否支持昇腾310P芯片

### DISCUSSION THREAD

**Discussion by xfg0913** [PaddlePaddle/PaddleOCR#13787] [Q&A]
计划在华为的Atlas 300I Duo卡上（采用昇腾310P芯片）运行PaddleOCR，进行文字识别和版面恢复。请问能否运行？如何安装部署？
谢谢大家！

**Comment by zyscodes** [comment]
我在昇腾910B上运行出现了问题，目前还没有解决。
![image](https://github.com/user-attachments/assets/c0c99be9-9b00-4cf7-976a-8e6325d87edd)
